src/languageservice/parser/jsonParser07.ts

```diff
@@ -1385,6 +1387,7 @@ function validate(
                 length: propertyNode.keyNode.length,
               },
               severity: DiagnosticSeverity.Warning,
+              code: ErrorCode.PropertyExpected,
```

Comment:
not sure if it wouldn't be better to unify `data.properties` to `data.values` for the incorrect property